source/configuring_domain/02_managing_legal_information/legals.rst

```diff
@@ -68,22 +68,68 @@ To specify the Terms and Conditions and the Privacy Policy of a portal, go to Co
 Licenses
 --------
 
-When configuring the metadata of a dataset in the Information tab, the license attached to that dataset should be defined, using a drop-down selection of predefined licenses. To define the licenses listed in the drop-down selection, go to Configuration > Legals. In this subsection of the back office menu, look for the "Licenses" area.
+When configuring the metadata for a dataset in the Information tab, it is important to define the license that specifies how others can use the data.
 
-.. image:: images/license_configuration.png
+Configure licenses
+~~~~~~~~~~~~~~~~~~
 
-1. Click the Add license button.
-2. In the Labels column, write the label of the license. The label will be displayed both in the drop-down selection in the back office, and in the Information tab of the published dataset in the front office. There must be one label per language available on the domain.
-3. (optional) In the URLs column, indicate the official website of the defined license, in order to find more information about that license. There can be one URL per language available on the domain.
-4. Click the Save button in the top right corner of the page.
+In the back office, you can configure the licenses available when adding the metadata for a dataset.
+
+1. From the left menu under Configuration, select Legals.
+2. Go to the Licenses area.
+   
+   .. image:: images/license_configuration.png
+
+3. Click the Add license button.
+4. (optional) In the URLs column, add the URL to the official website of the defined license to find more information about that license. You can only add one URL per language available on the domain.
+5. Click the Save button in the top right corner of the page.
 
 .. admonition:: Note
    :class: note
 
-   If there is no URL indicated for one of the languages, the platform will consider that there is indeed no URL attached to the license label for one language. The license label displayed in the front office will not be clickable at all when the portal is set on that language.
+   If you do not add the URL for a language, the license label displayed in the front office will not be clickable when the portal is set on that language.
 
 To delete a license, click on the |icon-trash| button.
 
+Default licenses
+~~~~~~~~~~~~~~~~
+
+By default, the following licenses are available:
+
+.. list-table::
+   :header-rows: 1
+
+   * * Label
+     * URL
+     * Description
+
+   * * Public domain
+     * `<https://en.wikipedia.org/wiki/Public_domain>`_
+     * The material is freely available. You can use it without any permission.
```

Comment:
I have an interrogation with the usage of "you": since we are talking about data, maybe we could use "Data can be used without any permission" or "It can be used without any permission" ?

Reply:
Let's use the passive form: "It can be used without any permission."
I'll change all occurrences.